pkg/cloudprovider/providers/azure/azure_vmss.go

```diff
@@ -585,6 +617,22 @@ func (ss *scaleSet) EnsureHostsInPool(serviceName string, nodes []*v1.Node, back
 		}
 	}
 	if !foundPool {
+		if ss.useStandardLoadBalancer() && len(newBackendPools) > 0 {
+			// Although standard load balancer supports backends from multiple vmss,
+			// the same network interface couldn't be added to more than one load balancer of
+			// the same type. Omit those nodes (e.g. masters) so Azure ARM won't complain
+			// about this.
+			backendPool := *newBackendPools[0].ID
```

Comment:
why not go through all newBackendPools here?

Reply:
We only need to check one because they are in same LB

Reply:
The pools is on path
VMSS -> Properties/NetworkProfile/networkInterfaceConfigurations/IPConfigurations/loadBalancerBackendAddressPools

Thus they can belong to different load balancers, right?

If master VMSS node is there and already with two LBs, the pool could be:
```
[
{ "id": "/*/Microsoft.Network/loadBalancers/master-internal/backendAddressPools/default" },
{ "id": "/*/Microsoft.Network/loadBalancers/master/backendAddressPools/default"  }
]
```

If not checking the second one, it may later add this node to another public LB and the call would fail. Please correct me if there's something I missed.

Reply:
Good catch. There is also same issue with standard vm codes. Will fix togather in a new commit.

Reply:
Fixed. @karataliu PTAL